### ISSUE TITLE
Properties on base classes not populated beyond 2 levels

### ISSUE THREAD

**Issue by cjdutoit** [Tynamix/ObjectFiller.NET#132]
**ISSUE**:  When a class derive from a class that derive from another class, then all the properties is not filled as expected.

```
public class BaseId
{
        public string Id { get; set; }
}

public class BaseWithAudit : BaseId
{
        public string CreatedBy { get; set; }
        public string UpdatedBy { get; set; }        
}

public class Person : BaseWithAudit
{
        public string Name { get; set; }
        public string LastName { get; set; }
}

```
Using the above classes to create a basic unit test, the test fails if the derived class have a base implementation of more than one level.

```
var filler = new Filler<Person>();
var x = filler.Create();

x.Name.Should().NotBeNullOrEmpty();
x.LastName.Should().NotBeNullOrEmpty();
x.CreatedBy.Should().NotBeNullOrEmpty();
x.UpdatedBy.Should().NotBeNullOrEmpty();
x.Id.Should().NotBeNullOrEmpty();
```
resulting in:
x.Name = "Bibirit"
x.LastName = "Kubuna"
x.CreatedBy= "Gucutazijaz"
x.UpdatedBy= "Fip"
**x.Id= ""**     ==> expected a value here

The same code just with one base class:
```
var filler = new Filler<BaseWithAudit>();
var x = filler.Create();

x.CreatedBy.Should().NotBeNullOrEmpty();
x.UpdatedBy.Should().NotBeNullOrEmpty();
x.Id.Should().NotBeNullOrEmpty();
```
All values filled as expected:
x.CreatedBy= "Bonotubirikid"
x.UpdatedBy= "Higevoka"
**x.Id= "Wuf"**   ==>value populated as expected

**Comment by endyprekpalaj** [comment]
Just wanted to confirm this. Having the same issue, and also works when inside level 2 inheritance.

**Comment by Tynamix** [comment]
Hey,
i reproduced and fixed it. Sorry for the looooooooooong delay. I quitted my Job as .NET Developer and always when i have to update ObjectFiller.NET i have to update my Visual Studio and .NET and so on. I have fixed the bug and will release it soon

**Comment by endyprekpalaj** [comment]
Great news, thank you for your time!


Greetings!
________________________________

**Comment by Tynamix** [comment]
Released in version 1.5.7. Please update the library!
